### ISSUE TITLE
Fedora instructions contain unneeded step 

### ISSUE THREAD

**Issue by hcal** [Tow-Boot/Tow-Boot#87]
You have a section under "Additionally, the EFI bootloader shim appears not to work under the conditions" with instructions for replacing BOOTAA64.EFI with grubaa64.efi.  That doesn't seem to be needed when installing Fedora on an nvme drive in a Pinebook Pro.  It works as is after ignoring the errors in the installer as described on the page.

**Comment by samueldr** [comment]
Indeed, I can confirm with a recent test (checking for this issue) that it is not needed anymore. The shim now seems to be able to correctly boot without doing any manual steps.
